Disable cabal citeproc, as we don't accept checking out git-repos during
build. It has to go into an extra package.

If it is needed please contact the maintainer of this package.

--- a/cabal.project
+++ b/cabal.project
@@ -3,12 +3,3 @@
 package pandoc
   flags: +embed_data_files -trypandoc
   ghc-options: -j +RTS -A64m -RTS
-
-package pandoc-citeproc
-  flags: +embed_data_files +bibutils -unicode_collation -test_citeproc -debug
-  ghc-options: -j +RTS -A64m -RTS
-
-source-repository-package
-    type: git
-    location: https://github.com/jgm/pandoc-citeproc
-    tag: 0.17
